Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -625,3 +626,20 @@ private func distinctRewards(_ rewards: [Reward]) -> [Reward] {
     return !rewardIds.contains(reward.id)
   }
 }
+
+// TODO: Remove this when implementing the API  [MBL-1851](https://kickstarter.atlassian.net/browse/MBL-1851)
```

Comment:
Moved this and changed the ticket number. It's no longer needed in `NoShippingPledgeViewModel`, but still needed here in `ManagePledgeViewModel`.